src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -9208,49 +9201,43 @@ class Program
     static ref R M5(ref R x, scoped ref R y) => ref x;
     static void Implicit()
     {
-        D1 dA = M2;
-        D2 d2 = M1;
+        D1 d1 = M2;
+        D2 d2 = M1; // 1
```

Comment:
it would be helpful to add these trailing comments on all the tests where the diagnostics changed. Also, it looks like the comments are incomplete in this test. I would expect comments for declarations of 'M3' and 'M5'.

Reply:
> I would expect comments for declarations of 'M3' and 'M5'.

I thought it would be clearer to have comments for the diagnostics related to `scoped` mismatch only.